_source/user-guide/kibana/log-patterns.md

```diff
@@ -3,37 +3,33 @@ layout: article
 title: Log patterns
 description: Log Patterns analyzes log messages and groups them according to detected patterns. We built Log Patterns to live alongside your logs in Kibana. You can see your patterns under the Patterns tab, and then click the Logs tab to return Kibana's familiar Discover view.
 permalink: /user-guide/kibana/log-patterns.html
-flags:
-  beta: true
-  logzio-plan: pro
+#flags:
+  #beta: true
+  #logzio-plan: pro
 tags:
   - log-patterns
 contributors:
   - danielberman
   - imnotashrimp
+  - shalper
 ---
 
-The Log Patterns feature dissects incoming logs and
-groups them according to detected patterns.
-As your logs come into Kibana, you'll see how they fit into existing patterns,
-in real time.
+Log Patterns runs advanced clustering algorithms to group logs by their frequency of occurrence to help you identify patterns in your logs. Use Patterns to quickly single out unique or unusual events from the mass of recurring and repetitive events.
 
-The Log Patterns view shows how often each pattern happens.
-So you can quickly identify unique or unusual events,
-as well as recurring and repetitive events.
+As your logs come into Kibana, you'll see how they fit into existing patterns,
+in near real time.
 
-You can use log patterns to surface logs you might have otherwise missed,
-or you can use it to identify unnecessary logs that you no longer want to ship to your account.
+Log patterns can help you surface logs you might have otherwise missed, logs that should be captured by [Optimizers]({{site.baseurl}}/user-guide/optimizers/configure-optimizers.html) and saved to a Timeless Account for long-term retention, or logs that you don't need to ship to your account and can be dropped using [drop filters]({{site.baseurl}}/user-guide/accounts/drop-filters/).
```

Comment:
The sentence about the optimizers is really confusing and I am not even sure what was the meaning here and why was it added